src/jimp2/wireworld/z8/datamangment/Electron.java

```diff
@@ -11,12 +13,12 @@
     private Orientation orientation;
 
 
-    /**
-     * @param position 
-     * @param orientation
-     */
     public Electron(Point position, Orientation orientation) {
-        // TODO implement here
+        this.position = position;
+        this.orientation = orientation;
+        name = "Electron";
+        cellsContainer = new CellsContainer(1,1);
```

Comment:
3x3* z czego środkowe to główka i jeden z bocznych to ogon. Ewentualnie 2x2 ale to trochę mniej wygodne. Puste komórki nie są wstawiane (są pomijane).